Source/WebCore/css/CSSQuadValue.h

```diff
@@ -37,10 +37,11 @@ class CSSQuadValue final : public CSSValue {
 
     String customCSSText() const;
     bool equals(const CSSQuadValue&) const;
+    bool canBeCoalesced() const;
 
 private:
     explicit CSSQuadValue(Quad);
-
+    bool m_coalesceIdenticalValues { true };
```

Comment:
Is this the best location padding-wise? It might add 7 bytes of padding before the Quad, or it might pack in with a refcount. Worth checking.